backend/src/main/java/fk/prof/backend/http/LeaderHttpVerticle.java

```diff
@@ -109,4 +118,24 @@ private void handlePutAssociation(RoutingContext context) {
     }
   }
 
+  private void handleGetWork(RoutingContext context) {
+    try {
+      String appId = context.request().getParam("appId");
+      String clusterId = context.request().getParam("clusterId");
+      String procName = context.request().getParam("procName");
+      Recorder.ProcessGroup processGroup = Recorder.ProcessGroup.newBuilder().setAppId(appId).setCluster(clusterId).setProcName(procName).build();
+
+      BackendDTO.WorkProfile workProfile = this.policyStore.get(processGroup);
```

Comment:
Fixed in commit https://github.com/Flipkart/fk-prof/commit/91da4676d1f6265fa210b6a2e5b9dbc94cd3aea2